cpp/include/cugraph/graph_functions.hpp

```diff
@@ -663,7 +663,9 @@ extract_induced_subgraphs(
  *
  * @tparam vertex_t Type of vertex identifiers. Needs to be an integral type.
  * @tparam edge_t Type of edge identifiers. Needs to be an integral type.
- * @tparam edge_type_t Type of edge type identifiers. Needs to be an integral type.
+ * @tparam weight_t Type of edge weight.  Needs to be floating point type
+ * @tparam edge_id_t Type of edge id.  Needs to be an integral type.
+ * @tparam edge_type_t Type of edge type.  Needs to be an integral type.
```

Comment:
Aren't we currently limiting edge type to be int32_t? Better add something like, `currently, we support only int32_t for edge_type_t`?

Reply:
Fixed in next push